features/field_handlers.feature

```diff
@@ -21,7 +21,9 @@ Feature: FieldHandlers
       | field_post_links     | Link 1 - http://example.com, Link 2 - http://example.com                         |
       | field_post_select    | One, Two                                                                         |
       | field_post_address   | country: BE - locality: Brussel - thoroughfare: Louisalaan 1 - postal_code: 1000 |
-    Then I should see "Page one"
+    Then I should see "Post title"
+    And I should see "PLACEHOLDER BODY"
+    And I should see "Page one"
     And I should see "Page two"
```

Comment:
Out of scope for this issue, but I noticed that two fields were not tested here, so I added them.